plugins/cost-insights/src/types/public.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export type {
+  Alert,
+  AlertCost,
+  AlertForm,
+  AlertOptions,
+  AlertSnoozeFormData,
+  AlertDismissFormData,
+  AlertDismissOption,
+  AlertDismissReason,
+  AlertSnoozeOption,
+  AlertStatus,
+} from './Alert';
+export { AlertDismissOptions, AlertSnoozeOptions } from './Alert';
+export type { ChangeStatistic } from './ChangeStatistic';
+export type { Cost, Trendline } from './Cost';
+export type { Currency, CurrencyType } from './Currency';
+export type { DateAggregation } from './DateAggregation';
+export type { Duration } from './Duration';
+export { DEFAULT_DATE_FORMAT } from './Duration';
+export type { Entity } from './Entity';
+export type { Group } from './Group';
+export type { Metric, MetricData } from './Metric';
+export type { Product } from './Product';
```

Comment:
We use `Maybe` and `Project` types internally which we're not exporting here.  They're simple enough to implement ourselves but wanted to make sure this was intentional

Reply:
Ah, good call outs. `Project` should be exported. With the exception of the workflow component, we use the graphql-generated `Maybe` internally which is compatible.  It was intentionally omitted but not included in the changeset 😱. 

Though `Project`, `Group`, etc. are all fairly simple types, I think they should still be part of our public exports since they pertain to our data model(s). I stop short at `Maybe` since its so generic, it wouldn't make sense for an individual plugin to support it as part of its public API. If anything, it'd be part of some core types offered by backstage and shared across plugins.